config/settings.yml

```diff
@@ -1439,10 +1439,20 @@ genisis:
   pass: bogus
 
 # Settings for connecting AFS Veteran Services
+# For locahost we can use the existing certs as long as we don't call out
 dgi:
+  # add med_api here? Will need to reach out to that team
   jwt:
-    public_key_path: modules/meb_api/spec/fixtures/dgi_public_test.pem
-    private_key_path: modules/meb_api/spec/fixtures/dgi_private_test.pem
+    public_key_path: ""
+    private_key_path: ""
+  vye:
+    jwt:
+      # May not need the public path
+      public_key_path: "/dsva-vagov/vets-api/#{Settings.vsp_environment}/dgib/jwt.key"
+      private_key_path: "/dsva-vagov/vets-api/#{Settings.vsp_environment}/dgib/jwt.crt"
+    vets:
+      url: "https://dgi-uat001-vaapi.np.afsp.io/vets-service/v1"
```

Comment:
Please use a fake url

Reply:
I believe we need that url to test with

Reply:
You can assign the url to settings.local.yml so that it's not pushed to prod

Reply:
We need to test this on staging (vs our local), so we're wondering about settings.staging.yml.  Is that what settings.staging.yml is for?